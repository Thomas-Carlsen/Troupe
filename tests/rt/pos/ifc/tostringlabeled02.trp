let val x = 42 raisedTo {alice} 
    val y = 11 raisedTo {bob} 
    val s = toStringL (x,y)
    val _ = printString ("The level of the string is " ^ s )
in printString "Observe how the PC-level and blocking-level are now tainted because of the string concatenation (and the usage of printString) above"
end
