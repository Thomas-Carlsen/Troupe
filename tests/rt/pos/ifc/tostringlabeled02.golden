2019-03-07T19:02:08.670Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
The level of the string is (42@{alice}%{}, 11@{bob}%{})@{}%{}
Observe how the PC-level and blocking-level are now tainted because of the string concatenation (and the usage of printString) above
>>> Main thread finished with value: ()@{}%{}
